frontends/open-discussions/webpack.config.prod.js

```diff
@@ -34,16 +34,17 @@ module.exports = Object.assign(prodConfig, {
   context: __dirname,
   mode:    "production",
   output:  {
-    path:               path.resolve("./static/bundles/"),
+    path:               path.resolve(__dirname, `../../static/bundles/`),
     filename:           "[name]-[chunkhash].js",
     chunkFilename:      "[id]-[chunkhash].js",
-    crossOriginLoading: "anonymous"
+    crossOriginLoading: "anonymous",
+    publicPath:         "./"
```

Comment:
@rhysyngsun Is this OK? I suspect this will probably change soon based on our django-webpack discussion... My impression is that we are currently changing the public path on the django side.

But... I'm still learning about some of this stuff and I don't really know what publicPath is for. I added this line because the webpack build was failing with:
```
Webpack5 Automatic publicPath is not supported in this browser
```
(https://stackoverflow.com/questions/64294706/webpack5-automatic-publicpath-is-not-supported-in-this-browser)